src/components/GoalSuggest.tsx

```diff
@@ -117,9 +118,9 @@ export const GoalSuggest = forwardRef<HTMLDivElement, GoalSuggestProps>(
                 value={value}
                 onChange={onSelectItem}
                 items={items}
-                visible={visible}
+                visible={visible && showSuggest}
```

Comment:
What the practice to rename props? Can we not to do this? `visible` > `visible` — it's clear

Reply:
`showSuggest` prop has changed after select any item fron suggest, this is not rename